tests/playwright/pages/editor-page.js

```diff
@@ -778,4 +794,34 @@ module.exports = class EditorPage extends BasePage {
 		await this.setSliderControlValue( selector + '_font_size', fontsize );
 		await this.page.locator( '.elementor-control-' + selector + '_typography .eicon-edit' ).click();
 	}
+
+	/*
+	* Checks for a stable UI state by comparing screenshots at intervals and expecting a match.
+	* Can be used to check for completed rendering. Useful to wait out animations before screenshots and expects.
+	* Should be less flaky than waitForLoadState( 'load' ) in editor where Ajax re-rendering is triggered.
+	*/
+	async isScreenShottable( locator, retries = 3, timeout = 500 ) {
```

Comment:
@davseve I like the proposal for broader naming. I've noticed that I started using this to check stability before other expects than Screenshots. Extracting this would be useful for FE as well.

Reply:
@Barcov Please send me a PM about this, because I have copied and pasted your function into the pro version as well :).

Reply:
Btw It would be so much better if we would have one testing codebase.

Reply:
@hein-obox & @davseve We'll need to consider this refactor in a new task next sprint.